demo/text-field-label-demos.html

```diff
@@ -0,0 +1,39 @@
+<dom-module id="text-field-label-demos">
+  <template>
+    <style include="vaadin-component-demo-shared-styles">
+      :host {
+        display: block;
+      }
+    </style>
+
+
+    <h3>Label suffix</h3>
+    <vaadin-demo-snippet id="text-field-label-demos-label-suffix">
+      <template preserve-content>
+        <vaadin-password-field id="password" label="New password">
+          <vaadin-button id="suggest" slot="label-suffix" theme="tertiary-inline">suggest</vaadin-button>
```

Comment:
Per MDN web docs, placing interactive content inside a `<label>` is a bad practice we should avoid.
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#Interactive_content

> Don't place interactive elements such as anchors or buttons inside a label. Doing so makes it difficult for people to activate the form input associated with the label.

Reply:
Thank you @web-padawan for the reference, I already paid attention to this and if you look closely there is no interactive content inside the `<label>` element: the slot is side-by-side the label, inside a container `<div>`.

Reply:
Thanks, missed that.